packages/components/tests/acceptance/components/hds/button-test.js

```diff
@@ -0,0 +1,23 @@
+import { module, test } from 'qunit';
+import { visit } from '@ember/test-helpers';
+import { setupApplicationTest } from 'dummy/tests/helpers';
+import { a11yAudit } from 'ember-a11y-testing/test-support';
+
+module('Acceptance | Component | hds/button', function (hooks) {
+  setupApplicationTest(hooks);
+
+  test('Components/button passes a11y automated checks', async function (assert) {
+    // if making any changes to this component, enable the rule and run it to check that changes are still compliant
+    let axeOptions = {
+      rules: {
```

Comment:
@MelSumner if we are disabling the color contrast tests for all the use cases, then no it doesn't make sense (I prefer to remove the failing use case at that point). I thought there was a way to disable a specific check only for an element (eg. via a DOM selector); hence the idea of an "exception" that I was suggesting.